devlab/Vagrantfile

```diff
@@ -1,39 +1,43 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby ts=2 sw=2 et sua= inex= :
 
-public_key_path = "#{ENV["HOME"]}/.ssh/id_rsa.pub"
+config_file = "config.ini"
+options = {}
+File.foreach(config_file) { |line|
+  option, value = line.split("=")
+  option = option.strip()
+  if !value.nil?
+    value = value.strip()
+  end
+  options[option] = value
```

Comment:
In this case you should update `options` only if `value != nil` too (move `options[option] = value` into `if` statement.